pants.ini

```diff
@@ -391,6 +391,8 @@ resolver_cache_dir: %(pants_bootstrapdir)s/python_cache/requirements
 
 
 [test.pytest]
+fast: false
+chroot: true
```

Comment:
I'm totally open to setting this to `false` for now with a `TODO` to switch to `true`. About 99% of unit tests work with `--chroot` and about 50% of targets. It might be better to default to this being `false` until we can get the integration test number higher. 

The downside of this being `true` is that blacklisted tests must be run with `--no-test-pytest-chroot` to pass. The benefit of this being `true` is that you don't have to remember to do `--test-pytest-chroot` for non-blacklisted tests.